references/classification/train.py

```diff
@@ -80,20 +80,21 @@ def _get_cache_path(filepath):
     cache_path = os.path.expanduser(cache_path)
     return cache_path
 
-
```

Comment:
Can you keep the newline for the linter?

Reply:
Done